### PR TITLE
50 47 nested and errors

### PR DESCRIPTION
**Pull request by CJGutz**
closes: #47
closes: #50

## Proposed changes
Show data with nested structures.

## Screenshots
image
image
image
image

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added
- [ ] Documentation have been written

### CODE REVIEW

File: src/app/components/vc-inspection/ParsedCredentialInfo.tsx [hunk 1]
```diff
@@ -22,6 +32,34 @@ function HLineWithText({ text }: { text: string }) {
   );
 }
 
+function convertNestedClaims(claims: Claim[]): LabeledValues[] {
```
Comment (thomsen85) [inline src/app/components/vc-inspection/ParsedCredentialInfo.tsx:35]: Docs?

File: src/inspector/calculatedAttributes/types.ts [hunk 2]
```diff
@@ -28,6 +29,11 @@ export function toSome<T>(obj: T): { kind: 'some'; value: T } {
 export function toNone(reason: string): { kind: 'none'; reason: string } {
   return { kind: 'none', reason };
 }
+
+export function isZodError(obj: unknown): obj is ZodError {
```
Comment (thomsen85) [inline src/inspector/calculatedAttributes/types.ts:33]: Docs

**Review by thomsen85 — APPROVED** [review]
Looks good, add some more docs and merge :D

File: src/app/components/vc-inspection/ParsedCredentialInfo.tsx (hunk 1 above)
Comment (CJGutz) [reply src/app/components/vc-inspection/ParsedCredentialInfo.tsx:42]: Yes

File: src/inspector/calculatedAttributes/types.ts (hunk 2 above)
Comment (CJGutz) [reply src/inspector/calculatedAttributes/types.ts:37]: Yes

**Pull request merged**